Tests/FluentAssertions.Specs/Exceptions/AsyncFunctionExceptionAssertionSpecs.cs

```diff
@@ -84,17 +84,21 @@ public async Task When_async_method_throws_an_empty_AggregateException_it_should
             await act2.Should().ThrowAsync<XunitException>();
         }
 
-        [UIFact]
-        public async Task When_async_method_throws_an_empty_AggregateException_on_UI_thread_it_should_fail()
+        [Collection("UIFacts")]
```

Comment:
I'm [pretty sure](https://xunit.net/docs/running-tests-in-parallel) that tests within the same class will never run in parallel. So moving all those tests in the same (partial) class should be enough.

Reply:
This change should also put tests across different classes in the same test collection to avoid _any_ `UIFact`s being run in parallel.